core/build.gradle

```
@@ -0,0 +1,25 @@
+plugins {
+    id 'java'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.71'
+    id 'org.jetbrains.kotlin.plugin.serialization' version '1.3.71'
+}
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+    jcenter()
+}
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    implementation "org.jetbrains.kotlinx:kotlinx-serialization-runtime:0.20.0" // JVM dependency
+    testCompile group: 'junit', name: 'junit', version: '4.12'
```

Unrelated, but low-hanging fruit; we could take the chance to update this deprecate configuration with
```suggestion
    testImplementation group: 'junit', name: 'junit', version: '4.12'
```